test/extensions/filters/http/ext_proc/ext_proc_integration_test.cc

```diff
@@ -86,21 +92,61 @@ class ExtProcIntegrationTest : public HttpIntegrationTest,
     setDownstreamProtocol(Http::CodecType::HTTP2);
   }
 
+  void modifyHttpVirtualHost(const std::function<void(VirtualHost&)>& cb) {
+    config_helper_.addConfigModifier([cb](envoy::config::bootstrap::v3::Bootstrap& bootstrap) {
```

Comment:
you should be able to use this method 

https://github.com/envoyproxy/envoy/blob/716ee8abc526d51f07ed6d3c2a5aa8a3b2481d9d/test/config/utility.cc#L1299

Then you needn't to find out the hcm by yourself.

Reply:
Thanks, that saves a bunch of code.